### ISSUE TITLE
Other device tabs

### ISSUE THREAD

**Issue by digitek007** [dragonofmercy/Chrome-Better-History#38]
First, love this extension, thank you. Only thing that forces me to disable/remove it constantly is lack of access to the "Tabs from Other Devices" feature of history, as in Chrome the history panel appears to be the only reliable place to see this.  Request this be added, or if an easy option to temporarily revert back to default history, this would be okay too.  Right now I've removed the extension to have constant access to other device tabs.

**Comment by DavidGretzschel** [comment]
Yes, that's very annoying and why I can't use this otherwise lovely extension :(

The trouble is that entering "chrome://history/syncedTabs" automatically redirects to "chrome://history" and shows the "Chrome Better History"-history.
The fix might be simply to disable that redirect?
Then the /syncedTabs part could be left at standard Chrome behaviour, the history itself could be "Better History".

**Comment by dondolarson** [comment]
I'm for it too! Please improve your beautiful extension.
